include/nps_uw_sensors_gazebo/nps_gazebo_ros_gpu_sonar_single_beam.hh

```diff
@@ -15,17 +15,20 @@
  *
 */
 
-#ifndef NPS_GAZEBO_ROS_GPU_SINGLE_BEAM_SONAR_H
-#define NPS_GAZEBO_ROS_GPU_SINGLE_BEAM_SONAR_H
 
-#include <string>
+#ifndef NPS_GAZEBO_ROS_GPU_SONAR_SINGLE_BEAM_H_
+#define NPS_GAZEBO_ROS_GPU_SONAR_SINGLE_BEAM_H_
+
```

Comment:
Please change the macro variable to reflect the new file name: NPS_GAZEBO_ROS_GPU_SONAR_SINGLE_BEAM_HH_